pt_bot.py

```diff
@@ -1,50 +1,75 @@
 import logging
 import re
-
 import requests
 import telegram
+from flask import Flask, request, Response, render_template
 from telegram import ChatAction
 from telegram.ext import Updater, CommandHandler, MessageHandler, Filters, ConversationHandler
-
 import pt_config
 import pt_error
 import pt_momo
 import pt_service
 from lotify_client import get_lotify_client
 from pt_entity import UserGoodInfo, GoodInfo
+import os
+
+template_dir = os.path.abspath('Templates')
+
+lotify_client = get_lotify_client()
 
 updater = Updater(token=pt_config.BOT_TOKEN, use_context=True)
 dispatcher = updater.dispatcher
 bot = telegram.Bot(token=pt_config.BOT_TOKEN)
 logger = logging.getLogger('Bot')
-lotify_client = get_lotify_client()
 
 UNTRACK = range(1)
 ADD_GOOD = range(1)
 
+logging.basicConfig(format='%(asctime)s - %(name)s - %(levelname)s - %(message)s',
+                    level=pt_config.LOGGING_LEVEL, force=True)
+app = Flask(__name__, template_folder=template_dir)
 
-def run():
-    if pt_config.TELEGRAM_BOT_MODE == 'polling':
-        bot_updater = Updater(token=pt_config.BOT_TOKEN, use_context=True)
-        bot_dispatcher = bot_updater.dispatcher
-        bot_updater.start_polling()
-    else:
-        import os
-        port = int(os.environ.get('PORT', '8443'))
-        bot_updater = Updater(pt_config.BOT_TOKEN)
 
-        bot_updater.start_webhook(listen="0.0.0.0",
-                                  port=port,
-                                  url_path=pt_config.BOT_TOKEN,
-                                  webhook_url=pt_config.WEBHOOK_URL + pt_config.BOT_TOKEN)
-        bot_dispatcher = bot_updater.dispatcher
+@app.route('/', methods=['GET'])
+def index():
+
+    # health check
+    return render_template('index.html')
+
```

Comment:
I think returning `HTTP 200` with the message `ok` is enough.

Reply:
fixed